pkg/manifests/manifests.go

```diff
@@ -2508,6 +2508,22 @@ func (f *Factory) ThanosQuerierDeployment(grpcTLS *v1.Secret, enableUserWorkload
 		d.Spec.Template.Spec.Volumes = append(d.Spec.Template.Spec.Volumes, volume)
 	}
 
+	if f.config.ThanosQuerierConfig.Resources != nil {
```

Comment:
So we just default to worker nodes?

Reply:
correct, that is what i synced with @brancz